guides/leave_space_mutation.md

```diff
@@ -0,0 +1,49 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'leave-space-mutation'
+is: 'experimental'
+---
+
+# Leave Space
+
+## Concepts
+
+The _leaveSpace_ mutation allows the user to leave a space. The mutation accepts a Space ID as an argument and makes the request on behalf of the calling user. Currently this API can only be used to reject an invitation to a space. Otherwise it will return an error.
```

Comment:
>Currently this API can only be used to reject an invitation to a space. Otherwise it will return an error.

Is this temporary?  The name of the mutation leads me to believe I can call this to leave an existing space I am a member of. If this is permanent then this as well as its counterpart `join` API should perhaps be named:
- AcceptInvitation
- RejectInvitation

Reply:
Yes, it's temporary. The plan is to extend to all space members before the API is moved to beta.

Reply:
we should make a note of the fact that it's temporary

Reply:
I hoped this implied that it could be extended later.

> Currently this API can only be used to reject an invitation to a space. 

Any suggestions. @jarusso

Reply:
Although in the future as you indicate one could use the leaveSpace mutation on any space they are a member of,  one will never be able to join a space other than via accepting an invitation.   This makes me wonder if we are better off distinguishing between 3 separate actions:

1. Accepting Invitation
2. Rejecting Invitation
3. Leaving a space (which means I was already a member of the space)

1 and 2 is what you are seeking now.  3 is actually different from 2.

Reply:
Retracting my comment above on  "one will never be able to join a space other than via accepting an invitation"